docs/source/torch.compiler_inductor_provenance.rst

```diff
@@ -0,0 +1,61 @@
+.. _torchinductor-provenance:
+
+TorchInductor and AOTInductor Provenance Tracking
+=================================================
+
+**Author:** `Shangdi Yu <https://github.com/yushangdi>`__ and `Rachel Guo <https://github.com/YUNQIUGUO>`__
```

Comment:
why are we adding this to pytorch/pytorch? This type of content traditionally lives in pytorch/tutorials: https://github.com/pytorch/tutorials